python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -0,0 +1,327 @@
+                     0 and 1 not valid for regression
+    bootstrap : boolean (default = True)
+                Control bootstrapping.
+                If set, each tree in the forest is built
+                on a bootstrapped sample with replacement.
+                If false, sampling without replacement is done.
+    bootstrap_features : boolean (default = False)
+                         Control bootstrapping for features.
+                         If features are drawn with or without replacement
+    rows_sample : float (default = 1.0)
+                  Ratio of dataset rows used while fitting each tree.
+    max_depth : int (default = -1)
+                Maximum tree depth. Unlimited (i.e, until leaves are pure),
+                if -1.
+    max_leaves : int (default = -1)
+                 Maximum leaf nodes per tree. Soft constraint. Unlimited,
+                 if -1.
+    max_features : int or float or string or None (default = 'auto')
+                   Ratio of number of features (columns) to consider
+                   per node split.
+                   If int then max_features/n_features.
+                   If float then max_features is a fraction.
+                   If 'auto' then max_features=n_features which is 1.0.
+                   If 'sqrt' then max_features=1/sqrt(n_features).
+                   If 'log2' then max_features=log2(n_features)/n_features.
+                   If None, then max_features=n_features which is 1.0.
+    n_bins :  int (default = 8)
+              Number of bins used by the split algorithm.
+    min_rows_per_node : int or float (default = 2)
+                        The minimum number of samples (rows) needed
+                        to split a node.
+                        If int then number of sample rows
+                        If float the min_rows_per_sample*n_rows
+    accuracy_metric : string (default = 'mse')
+                      Decides the metric used to evaluate the performance
+                      of the model.
+                      for median of abs error : 'median_ae'
+                      for mean of abs error : 'mean_ae'
+                      for mean square error' : 'mse'
+    
+    """
+    
+    def __init__(self, n_estimators=10, max_depth=-1, handle=None,
+                 max_features='auto', n_bins=8,
+                 split_algo=1, split_criterion=2,
+                 bootstrap=True, bootstrap_features=False,
+                 verbose=False, min_rows_per_node=2,
+                 rows_sample=1.0, max_leaves=-1,
+                 accuracy_metric='mse', min_samples_leaf=None,
+                 min_weight_fraction_leaf=None, n_jobs=None,
+                 max_leaf_nodes=None, min_impurity_decrease=None,
+                 min_impurity_split=None, oob_score=None,
+                 random_state=None, warm_start=None, class_weight=None,
+                 quantile_per_tree=False, criterion=None):
+
+
+        sklearn_params = {"criterion": criterion,
```

Comment:
Consolidate this checking into a common function that both regressor and classifier (and other models) can use.